### PR TITLE
Add console debug mode to increase output depending on verbosity

### PR DESCRIPTION
**Pull request by tgalopin**
This PR implements parts of the Symfony Monolog bridge in order to provide a debug output if the verbosity is high enough (-vvv).

### CODE REVIEW

**Comment by jderusse** [comment]
I wonder if all the output->writeln() could be remplaced by this new logger.

**Comment by tgalopin** [comment]
I thought about it, I'm not sure. What do you think? It would abstract the output and I have to admit I don't like very much having to add the output in the container.

File: src/Cli/Command/RequestCommand.php [hunk 1]
```diff
@@ -89,9 +89,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         // Certificate renewal
         if ($this->hasValidCertificate($domain, $alternativeNames)) {
+            $this->getCliLogger()->debug('Certificate found, executing renewal', [
```
Comment (jderusse) [inline src/Cli/Command/RequestCommand.php]: a lot of calls to `$this->getCliLogger()->...` may be optimize with a `$logger =  $this->getCliLogger()` on top of the method.

**Comment by jderusse** [comment]
Regarding the output, you may just replace every `$output->write` by a `$logger->info` and moving the default verbosity to maintain the BC.

**Pull request merged**